src/layouts/partials/Header.vue

```diff
@@ -36,7 +36,7 @@
         <div class="has-dropdown">
           <a
             aria-label="GitHub"
-            href="//github.com/gridsome/gridsome"
+            href="//github.com/gridsome"
```

Comment:
Not sure this change is needed? It links to the Gridsome source, which is intended.

Reply:
I follow WAVE's advice, "Adjacent links shoul'd go to the same URL”. So I change the link on the navigation and pointed it to the GitHub organization

Reply:
I understand, but I don't think that is really an issue in this case - IMHO there isn't really any point in linking to the GitHub org, and then making the user perform another link to get to where they would expect the link to take them anyway.

But perhaps that is just me, because I use it a lot to jump straight to the repo 🙃 perhaps @TerabyteTiger could give his opinion...

Reply:
In my initial review I misunderstood what this link was doing and assumed it was a typo fix 😅 

I think it makes sense to not have 2 links next to each other go to the same place, but what I would probably do is either 

1. reword the link in the flyout to indicate that it will go the the Gridsome user-level account instead of to the gridsome/gridsome repo
2. Add a link above the first flyout option to go to the Gridsome user-level account, then leave the existing link alone so that it's still clear that you can get to the gridsome/gridsome repo incase it isn't clear where the Github Icon & Version Number will take you when clicked (which I don't think it is - tbh I assumed it would take me to the gridsome.org repo 😅)

Reply:
Good suggestions - I had also misunderstood, I had forgotten there was another exact same link in the drop-down! I presumed @kecrily meant adjacent links in the same nav bar. 
In which case, I rescind my comments, and think this change does make sense 👍🙂